server/src/lib/api/api.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import express from 'express';
+import modelFinder from '../middleware/modelFinder';
+
+const router = express.Router();
+router.param('model', modelFinder);
+
+router.post('/api/v1/:model', (req, res, next) => {
+  let document = new req.model(req.body);

```

Comment:
document is specific to MongoDB so creating a more general name would be best. Something like 'record' or 'data' would be better.